app/domains/fluggastrechte/formular/stringReplacements.ts

```diff
@@ -185,3 +185,16 @@ export const hasBothAirportsPartnerCourts = (context: FluggastrechtContext) => {
       objectKeysNonEmpty(endAirport, ["zipCodePilotCourt"]),
   };
 };
+
+export const getResponsibleAirportForCourt = (
+  context: FluggastrechtContext,
+) => {
+  const startAirport = getAirportByIataCode(context.startAirport ?? "");
+  const isStartAirportResponsible =
+    (startAirport?.zipCodePilotCourt?.length ?? 0) > 0;
```

Comment:
```suggestion
  const startAirport = getAirportByIataCode(context.startAirport);
  const isStartAirportResponsible =
    startAirport && startAirport.zipCodePilotCourt.length > 0;
```

I just tried this and the test passed, what do you think? much easier to understand

Reply:
good idea :)